sources/packages/backend/libs/integrations/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt.processing.service.ts

```diff
@@ -220,18 +219,17 @@ export class DisbursementReceiptProcessingService {
           fileName: disbursementFileName,
         },
       );
-      result.processSummary.push(
+      processSummary.info(
         "Provincial daily disbursement CSV report generated.",
       );
       this.logger.log(
         "Completed provincial daily disbursement report generation.",
       );
-    } catch (error) {
-      this.logger.error(error);
-      result.errorsSummary.push(
-        "Error while generating provincial daily disbursement CSV report file.",
-      );
-      result.errorsSummary.push(error);
+    } catch (error: unknown) {
+      const errorMessage =
+        "Error while generating provincial daily disbursement CSV report file.";
+      this.logger.error(errorMessage, error);
+      processSummary.error(errorMessage, error);
```

Comment:
Should the retry happen when the report generation alone fail? - let me know if I am missing something here.

Reply:
Is there a reason not to retry? The import should be able to be executed twice and the report generation is part of the process, hence if it fails it would not need manual intervention.
pleas let me know if we need to discuss this further.